tutorials/setup.md

```diff
@@ -341,6 +339,11 @@ default_sysctls = [
 ]
 ```
 
+One optional sysctl you might consider, if you want ping to work within
+contianers and your kernel supports it is setting
+
+	"net.ipv4.ping_group_range = 0   429296728",
```

Comment:
what's this magic number?

Reply:
I believe this is the max number of gids on a system.

Reply:
2147483647?

Reply:
Ok, I will change, but I thought I found that number from the internet.  Also see a spelling mistake above.